api/src/pcapi/core/fraud/api/ubble.py

```diff
@@ -6,6 +7,9 @@
 from pcapi.models import db
 
 
+UBBLE_TEST_EMAIL_RE = re.compile(r"^.+\+ubble_test@.+$")
```

Comment:
La stratégie d'accepter tout email dans la regex `^.+\+ubble_test@.+$` me paraît risquée... Si je mets patrick.rouzet+ubble_test@gmail.com (google accepte tous les +... comme alias par défaut), j'aurai un compte qui court-circuite Ubble avec mon mail ; facile à voir c'est open-source, il suffit que quelqu'un trouve l'astuce sur github.
Ou alors tu as juste oublié de conditionné à `not settings.IS_PROD` plus loin, mais à première vue ça passe partout

Reply:
bien vu